easybuild/easyconfigs/l/Libint/Libint-2.0.3-foss-2018b.eb

```diff
@@ -0,0 +1,22 @@
+name = 'Libint'
+version = '2.0.3'
+
+homepage = 'https://sourceforge.net/p/libint'
+description = """Libint library is used to evaluate the traditional (electron repulsion) and certain novel two-body
+ matrix elements (integrals) over Cartesian Gaussian functions used in modern atomic and molecular theory."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+toolchainopts = {'pic': True}
+
+source_urls = ['http://downloads.sourceforge.net/project/libint/libint-for-mpqc']
+sources = ['libint-%(version)s-stable.tgz']
+# Remove some problematic (and not very useful) FMA header code that does not compile with GCC.
+# This is done with a patch, because it also affects anything that links against Libint.
+# This way, FMA compilert optimization can remain enabled.
+patches = ['Libint-2.0.3-stable-fma.patch']
```

Comment:
@tovrstra Please use `%(version)s` rather than hardcoding the `2.0.3`:

```python
patches = ['Libint-%(version)s-stable-fma.patch']

Reply:
Done